communication.py

```diff
@@ -75,73 +75,83 @@ def pckParser(self, line):
         # header = line[0] + line[1]
         length = line[2]
         length = int.from_bytes(length, "little", signed=False)
-
-        takim_no = line[3] + line[4]
-        takim_no = int.from_bytes(takim_no, "little", signed=False)
-        takim_no = takim_no * 8
-
-        paket_no = line[5] + line[6]
+        paket_no = line[4] + line[5]
```

Comment:
where is third byte? all of the numbers below here are written as one more than they should be. fix it.